File: Day8_RemoveOddPlaceCharacters/main.java
public class TechnologyEncroyable
{
	public static void main(String[] args) {
		String s = "Technology Encroyable";
		String r = "";
		for(int i = 0;i<s.length();i++){
		    if(i%2==0){
		        r += s.charAt(i);
		    }
		}
	    System.out.println(r);
	}
}

File: Day8_RemoveOddPlaceCharacters/main.cs
using System;
class TechnologyEncroyable {
  static void Main() {
    string org = "Technology Encroyable";
    string newStr = "";
    for(int i = 0;i<org.Length;i++){
        if(i%2==1)
           continue;
        newStr += org[i];   
        
    }
    Console.WriteLine(newStr);
  }
}
